### DISCUSSION TITLE
Markdown in Model Posts

### DISCUSSION THREAD

**Discussion by Janca** [civitai/civitai#105] [Ideas]
It would be great to be able to include a small subset of markdown in the main content and about model section. Such as code blocks, inline code styling and headers.

i.e:
\# Post Header
Some introductory information about model

\## Usage
Primary trigger word is \`sks_trigger\` but you may also use:
\```
a code, block, of, other, tags, words
\```

**Comment by manuelurenah** [comment]
Hey @Janca we have just made a new release adding support for markdown in the model description, now you'll be able to use markdown syntax to document your model properly. Hope this helps!

**Comment by Janca** [comment]
I see the new options in the editor, but they do not correctly render. I also tried using images, which previewed correctly in the editor, but live, wrapped with text, instead of being a block element.

In the editor
![preview](https://i.imgur.com/dRe106h.png)

Live
![preview2](https://i.imgur.com/bBNBkuv.png)

**Reply by JustMaier** [reply]
We're on it :)

**Reply by manuelurenah** [reply]
Should be good now, please check your model again and let us know if you're still running into the same issue

**Comment by Janca** [comment]
Sorry, but it's still broken,

Editor
![editor](https://i.imgur.com/gWauytI.png)

Live
![live](https://i.imgur.com/zDCjvmM.png)

Going back to editor after saving
![redit](https://i.imgur.com/j8Pxbwp.png)

**Reply by manuelurenah** [reply]
I see, the heading styles are not applying. I just did another release, now you should be able apply them correctly. Sorry for last time!

**Comment by Janca** [comment]
No, its just not working. Same as before. Previewed correctly in the editor, but live view was the exact same results, and then upon returning to the editor, all styles were lost. I also cleared my browser cache, and viewed not signed in to double check things, but same results.

**Reply by Janca** [reply]
And now it is working, excellent! Not sure if you made changes, or something was cached on the edge, but I have headers!

**Reply by JustMaier** [reply]
Thanks for working with us on this. We just reworked our editor this week and had to work out the kinks. Thanks for being quick with the feedback so we could make sure it was working in a live environment.
